ignite/distributed/comp_models/base.py

```diff
@@ -264,11 +279,11 @@ def broadcast(
         return tensor
 
     @abstractmethod
-    def _do_all_reduce(self, tensor: torch.Tensor, op: str = "SUM") -> torch.Tensor:
+    def _do_all_reduce(self, tensor: torch.Tensor, group: Optional[Any] = None, op: str = "SUM") -> torch.Tensor:
```

Comment:
```suggestion
    def _do_all_reduce(self, tensor: torch.Tensor, op: str = "SUM", group: Optional[Any] = None) -> torch.Tensor:
```
Let's make the change less breaking